Composer/packages/client/src/recoilModel/selectors/skills.ts

```diff
@@ -36,10 +45,27 @@ export const skillsStateSelector = selector({
           manifest,
           name: botName,
           remote: isRemote,
+          manifestId,
+          location,
         };
       }
       return result;
     }, {});
     return skills;
   },
 });
+
+export const skillIdByProjectIdSelector = selector({
```

Comment:
Do we need an additional selector ? `skillsStateSelector` seems like would solve your needs right? In addition, I think it makes sense to move the skillsProjectIdSelector which in inside selectors/project to selectors/skills as it fits in this file i think.

Reply:
I moved `skillsProjectIdSelector ` into `selectors/skills`.
This additional selector intent to found skillNameIdentifier by skillId, the existed `skillsStateSelector ` are identify by skillNameIdentifier. All I have here is skillId, add this selector would be more convenient.